client/my-sites/plugins/plugin-activate-toggle/index.jsx

```diff
@@ -111,7 +111,6 @@ export class PluginActivateToggle extends Component {
 			<PluginAction
 				disabled={ disabled }
 				className="plugin-activate-toggle"
-				label={ translate( 'Active', { context: 'plugin status' } ) }
```

Comment:
Can we use `aria-label` instead of completely removing any text that would help with accessibility?

Reply:
Now this keeps the label attribute and we pass as aria-label inside the component:  36aeab24044da59454aabc46a6f36caa963c0809